modules/nf-core/plink/genome/tests/main.nf.test

```diff
@@ -0,0 +1,38 @@
+nextflow_process {
+
+    name "Test Process PLINK_GENOME"
+    script "../main.nf"
+    process "PLINK_GENOME"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "plink"
+    tag "plink/genome"
+
+    
+    test("plink - genome") {
+        when {
+            params {
+                outdir = "test"
+            }
```

Comment:
```suggestion
```

Reply:
Thank you, I will have to find documentation for this as I am new to all this.